packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/parse-file-from-string.ts

```diff
@@ -14,11 +14,18 @@ type ParsedFile = {
 export function parseFileFromString(fileAsString: string): ParsedFile {
   let decodedBytes: Buffer;
 
-  try {
-    decodedBytes = Buffer.from(fileAsString, "base64");
-  } catch (ex) {
-    log("Got a non-base64 document! Using original fileAsString content for decodedBytes");
+  // with MTOM we now have non b64 encoded files in the body of the response.
+  const firstSixBytesBuffer = Buffer.from(fileAsString.slice(0, 6));
+  if (isXMLContentType(firstSixBytesBuffer)) {
     decodedBytes = Buffer.from(fileAsString.trim());
+  } else {
+    try {
+      decodedBytes = Buffer.from(fileAsString, "base64");
+    } catch (ex) {
+      // this doesnt really work. Didnt catch the case of plain xml
+      log("Got a non-base64 document! Using original fileAsString content for decodedBytes");
+      decodedBytes = Buffer.from(fileAsString.trim());
```

Comment:
I am not exactly sure why, but this code doesn't do what its supposed to and catch when the bytes are not b64. 
Will address in follow on ticket
https://github.com/metriport/metriport-internal/issues/1667

Reply:
But can this be merged/shipped w/o that fix?